embedded/tbtree/tbtree.go

```diff
@@ -1211,7 +1261,7 @@ func (t *TBtree) Compact() (uint64, error) {
 		return 0, ErrCompactionThresholdNotReached
 	}
 
-	_, _, err := t.flushTree(false)
+	_, _, err := t.flushTree(0, false)
```

Comment:
Shouldn't this be 
```suggestion
	_, _, err := t.flushTree(100, false)
```
?

Reply:
after compaction, current nLog will be physically deleted anyway

Reply:
OK, I thought we don't have the old-style compaction anymore.